.travis.yml

```diff
@@ -1,18 +1,26 @@
-language: php
-
 sudo: false
 
+language: php
+
 cache:
-  directory:
+  directories:
     - $HOME/.composer/cache
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7
-  - hhvm
+    - 7
+    - hhvm
+
+install:
+    - composer self-update
```

Comment:
@FabioBatSilva 

How about updating `composer` itself in the `before_install` section?

:information_desk_person: For reference see https://docs.travis-ci.com/user/customizing-the-build/#The-Build-Lifecycle.